### PR TITLE
[WIP] Update image: cita run and cita create config.

### PR DESCRIPTION
**Pull request by kaikai1024**
#### Description of the Change

Split cita_run image in env.sh:
* new cita run
* cita create config

#### Benefits
* Create cita config is independent.
* for #243 

#### Verification Process

Check two containers after create cita config and setup node.

#### Applicable Issues

#270

### CODE REVIEW

File: scripts/env.sh [hunk 1]
```diff
@@ -27,6 +27,11 @@ WORKDIR=/opt/cita-run
 USER_ID=`id -u $USER`
 USER_NAME="user"
 
+if [[ "${1}" =~ "create_cita_config" ]]; then
```
Comment (yangby-cryptape) [inline scripts/env.sh]: Why use `=~` ?

File: scripts/env.sh [hunk 2]
```diff
@@ -27,6 +27,11 @@ WORKDIR=/opt/cita-run
...
+    CONTAINER_NAME="cita_config${RELEASE_DIR//\//_}"
```
Comment (yangby-cryptape) [inline scripts/env.sh]: `"cita_config"` and `"cita_run"` should be a variable before define `CONTAINER_NAME`.
So, we can write `${RELEASE_DIR//\//_}` only once.

File: scripts/env.sh [hunk 3]
```diff
@@ -27,6 +27,11 @@ WORKDIR=/opt/cita-run
...
+    DOCKER_IMAGE="cita/cita-run:config-ubuntu-18.04-20190228"
```
Comment (yangby-cryptape) [inline scripts/env.sh]: May be `ubuntu-18.04-20190228` can be a variable.
```bash
ID="ubuntu-18.04-20190228"
NAME="cita/cita-run"
if [ ..omit .. ]; then
    VERSION="${ID}"
else
    VERSION="config-${ID}"
fi
DOCKER_IMAGE="${NAME}:${VERSION}"
```

**Comment by kaikai1024** [comment]
> I did not check the build scripts for docker `cita-run`.
> 
> But I think `cita/cita-run:config-${id}` should base on `cita/cita-run:${id}`.
> They should have same layers, and `cita/cita-run:config-${id}` has one more layer than `cita/cita-run:${id}`.

That's my fault. 
It should be `cita:cita-run` & `cita: cita-config`, I'll fix that.

File: scripts/env.sh (hunk 1 above)
Comment (kaikai1024) [reply scripts/env.sh]: For the future that If we refactor the create_cita_config.

More detail:
I want to use the same `env.sh`, so i need to differentiate two image type.
if `./env.sh *create_cita_config* xx`, then will use cita-config image.

File: scripts/env.sh (hunk 3 above)
Comment (kaikai1024) [reply scripts/env.sh]: get

File: scripts/env.sh (hunk 2 above)
Comment (kaikai1024) [reply scripts/env.sh]: :clap:

**Comment by kaikai1024** [comment]
Later. TBD
#283

**Pull request closed without merging**